site/jest.setup.ts

```diff
@@ -48,9 +48,7 @@ global.ResizeObserver = require("resize-observer-polyfill");
 // Polyfill the getRandomValues that is used on utils/random.ts
 Object.defineProperty(global.self, "crypto", {
 	value: {
-		getRandomValues: function (buffer: Buffer) {
-			return crypto.randomFillSync(buffer);
-		},
+		getRandomValues: (b: NodeJS.ArrayBufferView) => crypto.randomFillSync(b),
```

Comment:
you could even take this a step further and just say...

```suggestion
		getRandomValues: crypto.randomFillSync,
```